main.go

```diff
@@ -20,25 +19,41 @@ import (
 )
 
 type SlackStatus struct {
-	StatusText  string `json:"status_text"`
-	StatusEmoji string `json:"status_emoji"`
+	StatusText  string `mapstructure:"status_text" json:"status_text"`
+	StatusEmoji string `mapstructure:"status_emoji" json:"status_emoji"`
 }
 
-type slackAccount struct {
-	Name            string
-	Token           string
-	MeetingStatus   *SlackStatus
-	NoMeetingStatus *SlackStatus
+type Account struct {
+	Name            string      `mapstructure:"name"`
+	Token           string      `mapstructure:"token"`
+	MeetingStatus   SlackStatus `mapstructure:"meetingStatus"`
+	NoMeetingStatus SlackStatus `mapstructure:"noMeetingStatus"`
```

Comment:
Any reason to not use pointers here so you can do nil-checks instead of `== (SlackStatus{})`?

Reply:
This was primarily to make it easier to dump the `Config` struct to the console. However, I have changed this and added `Stringer` functions to handle the formatting.